tests/unit/helpers.js

```diff
@@ -37,23 +38,29 @@ import {
 } from 'core/constants';
 import { ErrorHandler } from 'core/errorHandler';
 import { makeI18n } from 'core/i18n/utils';
+import { createInternalAddon } from 'core/reducers/addons';
 import {
   autocompleteLoad,
   autocompleteStart,
 } from 'core/reducers/autocomplete';
 import { searchLoad, searchStart } from 'core/reducers/search';
 import { selectUIState } from 'core/reducers/uiState';
 import { loadCurrentUserAccount } from 'amo/reducers/users';
+import { createInternalVersion } from 'core/reducers/versions';
 import { createUIStateMapper, mergeUIStateProps } from 'core/withUIState';
 import { addQueryParamsToHistory } from 'core/utils';
 
 export const sampleUserAgent =
   'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:40.0) Gecko/20100101 Firefox/40.1';
 export const sampleUserAgentParsed = UAParser(sampleUserAgent);
 
+export const createLocalizedString = (string, lang = 'en-US') => {
```

Comment:
Instead of having `'en-US'` defined in different functions, maybe we could introduce a constant (`DEFAULT_LANG` or `DEFAULT_LANG_IN_TESTS`) in this file? We could use `'en-US'` or `config.get('defaultLang')` as a value.

Reply:
Yeah, I like this idea. I'll change it.